openstack/dis/v3/streams/requests.go

```diff
@@ -0,0 +1,32 @@
+package streams
+
+import (
+	"github.com/chnsz/golangsdk"
+)
+
+type UpdateOpts struct {
+	StreamName                 string `json:"stream_name" required:"true"`
+	DataDuration               *int   `json:"data_duration,omitempty"`
+	DataType                   string `json:"data_type,omitempty"`
+	DataSchema                 string `json:"data_schema,omitempty"`
+	AutoScaleEnabled           bool   `json:"auto_scale_enabled,omitempty"`
```

Comment:
should be `*bool`

Reply:
always has value ,default value is false . keep it as it is

Reply:
fix as suggested